modules/appeals_api/app/models/appeals_api/non_veteran_claimant.rb

```diff
@@ -4,7 +4,7 @@ module AppealsApi
   class NonVeteranClaimant
     def initialize(auth_headers:, form_data:)
       @auth_headers = auth_headers
```

Comment:
I think you'll have a decent number of failures if you pass in `nil` for auth_headers, since there isn't an `|| {}` fallback to make methods like `first_name` work if they're `nil`.

I think this should still be `@auth_headers = auth_headers || {}`